src/app/dashboard/page.tsx

```diff
@@ -139,150 +162,154 @@ const Home: NextPage = () => {
+                </chakra.div>
+              ) : (
+                <Skeleton
+                  h={{ xl: '6', base: '4' }}
+                  w={{ xl: '32', base: '20' }}
+                  borderRadius="full"
                 />
-              )
-            })}
-          </Flex>
+              )}
+              <PriceDetails graphData={graphData} position="HIGHEST" />
+            </Flex>
+            <Flex
+              mt="27px"
+              sx={{
+                '.recharts-surface, .recharts-wrapper': {
+                  w: 'full',
+                },
+                '.recharts-tooltip-cursor, .recharts-area-curve': {
+                  color: 'brandText',
+                  stroke: 'currentColor',
+                },
+                '.gradientStart': {
+                  color: 'brandText',
+                  _dark: {
+                    color: 'rgba(255, 26, 136, 0.2)',
+                  },
+                },
+                '.gradientStop': {
+                  color: 'white',
+                  _dark: {
+                    color: 'transparent',
+                  },
+                },
+              }}
+            >
+              <ResponsiveContainer width="100%" height={120}>
+                {graphData !== undefined ? (
+                  <AreaChart data={graphData}>
+                    <Tooltip content={<CustomTooltip />} />
+                    <GraphLine />
+                    <Area
+                      className="area"
+                      activeDot={{ r: 4 }}
+                      type="monotone"
+                      dataKey="amt"
+                      stroke="#FF1A88"
+                      fillOpacity={1}
+                      fill="url(#colorUv)"
+                    />
+                  </AreaChart>
+                ) : (
+                  <Flex
+                    direction="column"
+                    alignItems="center"
+                    justifyContent="center"
+                  >
+                    <Spinner
+                      thickness="4px"
+                      speed="0.4s"
+                      color="graphSpinnerColor"
+                      emptyColor="graphSpinnerEmptyColor"
+                      size={{ xl: 'xl', base: 'md' }}
+                    />
+                    <Text mt="5" color="tooltipColor">
+                      Fetching chart data
+                    </Text>
+                  </Flex>
+                )}
+              </ResponsiveContainer>
+            </Flex>
+            <Flex>
+              <PriceDetails graphData={graphData} position="LOWEST" />
+            </Flex>
+            <GraphPeriodWrapper getRootProps={getRootProps}>
+              {GRAPH_PERIODS.map((btn) => {
+                return (
+                  <GraphPeriodButton
+                    key={btn.period}
+                    label={btn.label}
+                    {...getRadioProps({ value: btn.period })}
+                  />
+                )
+              })}
```

Comment:
The graph component from line 167- 288 can be reduced using the  <GraphComponent/>